openreview/conference/templates/areachairWebfield.js

```diff
@@ -803,8 +806,18 @@ var registerEventHandlers = function() {
   $('#group-container').on('click', 'a.collapse-btn', function(e) {
     if ($(this).text() === 'Show reviewers') {
       $(this).text('Hide reviewers');
+      var paperId=$(this).attr('href').split("-")[0].replace("#","");
+      var paperNumber=reviewerSummaryMap[Object.keys(reviewerSummaryMap).filter(p=>reviewerSummaryMap[p].noteId===paperId)].paperNumber;
+      updateReviewerContainer(paperNumber,true);//render empty dropdown as placeholder for visual consistency
+      var allReviewersWithConflictP=Webfield.get('/edges',{head:paperId,invitation:REVIEWER_GROUP_WITH_CONFLICT})
+      .then(result=>{
+        reviewerWithConflictForThisPaper=result.edges.map(p=>p.tail);
```

Comment:
I think instead of being a global variable, this can be passed to `updateReviewerContainer`